File: src/FacilityManagement.Core/AggregatesModel/UserAggregate/UserDto.cs
using System;

namespace FacilityManagement.Core
{
    public class UserDto
    {
        public Guid? UserId { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public byte[] Salt { get; set; }
        public List<Role> Roles { get; set; }
        public List<Profile> Profiles { get; set; }
    }
}

File: src/FacilityManagement.App/src/app/@api/models/user-dto.ts
/* tslint:disable */
import { ProfileDto } from './profile-dto';
import { RoleDto } from './role-dto';
export interface UserDto {
  password?: string;
  profiles?: Array<ProfileDto>;
  roles?: Array<RoleDto>;
  salt?: ArrayBuffer;
  userId?: string;
  userName?: string;
}
